pytrip/ctx.py

```diff
@@ -116,6 +116,7 @@ def create_dicom(self):
             pixel_array = np.zeros((_ds.Rows, _ds.Columns), dtype=self.pydata_type)
             pixel_array[:][:] = self.cube[i][:][:]
             _ds.PixelData = pixel_array.tostring()
+            _ds.PixelSpacing = [str(self.pixel_size),str(self.pixel_size)]
```

Comment:
PixelSpacing was already added to generated DICOMs in pytrip 1.3.1, see that without your change it gets inside the DICOM:

```
# grzanka at nz61-6 in ~/PycharmProjects/pytrip on git:master ✖︎ [20:18:32]
→ ~/.local/bin/trip2dicom -V                                                                                                                                         ~/PycharmProjects/pytrip
1.3.1

# grzanka at nz61-6 in ~/PycharmProjects/pytrip on git:master ✖︎ [20:18:37]
→ cat xio2.hed | grep pixel                                                                                                                                          ~/PycharmProjects/pytrip
pixel_size 0.7031

# grzanka at nz61-6 in ~/PycharmProjects/pytrip on git:master ✖︎ [20:18:45]
→ ~/.local/bin/trip2dicom xio2.hed xio2-dicom/                                                                                                                       ~/PycharmProjects/pytrip
Convert CT images
Convert structures
# grzanka at nz61-6 in ~/PycharmProjects/pytrip on git:master ✖︎ [20:18:53]
→ dcmdump xio2-dicom/CT.PYTRIP.100.dcm| grep Pixel                                                                                                                   ~/PycharmProjects/pytrip
(0028,0002) US 1                                        #   2, 1 SamplesPerPixel
(0028,0030) DS [0.7031\0.7031]                          #  14, 2 PixelSpacing
(0028,0103) US 1                                        #   2, 1 PixelRepresentation
(7fe0,0010) OW fbd8\fbd8\fbd8\fbd8\fbd8\fbd8\fbd8\fbd8\fbd8\fbd8\fbd8\fbd8\fbd8... # 524288, 1 PixelData
```

I do not see the point of adding it here.

It was already added in:
https://github.com/pytrip/pytrip/blob/07553ac53785cea543b25b565fc8e061e862e78b/pytrip/cube.py#L481
```   ds.PixelSpacing = [self.pixel_size, self.pixel_size]```

then copied in https://github.com/pytrip/pytrip/blob/07553ac53785cea543b25b565fc8e061e862e78b/pytrip/ctx.py#L108

```
_ds = copy.deepcopy(ds)
```